### PR TITLE
Implement Remote Configuration for Survey Screen

### PR DESCRIPTION
**Pull request by AndriiHorishniiMOC**
Implement Remote Configuration for Survey Screen

MOB-1665

photo_2022-12-02 23 36 41

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/survey/SurveyView.kt [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    private var controller: SurveyContract.Controller? = null
+
+    private var uiTheme: UiTheme by Delegates.notNull()
+
+    private val surveyTheme: SurveyTheme? by lazy {
+        Dependencies.getGliaThemeManager().theme?.surveyTheme
+    }
+
+    private val binding: SurveyViewBinding by lazy {
+        SurveyViewBinding.inflate(LayoutInflater.from(this.context), this)
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/survey/SurveyView.kt:55]: you can replace LayoutInflater.from with layoutinflater variable from ViewExtensions

Comment (AndriiHorishniiMOC) [reply widgetssdk/src/main/java/com/glia/widgets/survey/SurveyView.kt]: done

**Review by andrews-moc — APPROVED** [review]

**Review by gugalo — APPROVED** [review]

**Comment by dukhovnyi** [comment]
!fixup

**Comment by AndriiHorishniiMOC** [comment]
!merge

**Pull request merged**